docs/governance/overview.md

```diff
@@ -1,30 +1,25 @@
 # <img class="dcr-icon" src="/img/dcr-icons/Governance.svg" /> Introduction to Decred Governance
+	
+Decred is a Decentralized Autonomous Organization (DAO). Changes to the system, for example consensus rule changes, can happen in a decentralized manner. Decred’s governance is based on the principle of ticket-holder voting. To participate in governance, users must time-lock DCR to purchase tickets. There are multiple things ticket holders can vote on. Some of them are on-chain, and some of them are off-chain.
 
----
+# On-chain Voting
```

Comment:
On other pages we only have a single first-level header. Please make this header second level, and adjust the other headers accordingly

Reply:
Also Capital "Chain"